### PR TITLE
Prevent caching for JSON responses in http utils (close #105)

### LINKED ISSUE #105: Prevent caching for JSON responses in http utils

### PR DESCRIPTION
**Pull request by churkin**
\сс @inikulin @LavrovArtem

### CODE REVIEW

File: src/utils/http.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ export function respondWithJSON (res, data, skipContentType) {
     if (!skipContentType)
         res.setHeader('content-type', 'application/json');
 
+    // NOTE: GH-105
+    res.setHeader('cache-control', 'no-cache');
```
Comment (inikulin) [inline src/utils/http.js]: It should be `'no-cache, no-store, must-revalidate'`

**Comment by inikulin** [comment]
\r-

**Comment by inikulin** [comment]
lgtm

**Comment by LavrovArtem** [comment]
lgtm

**Pull request merged**